src/lxc/lxc_user_nic.c

```diff
@@ -1007,6 +1007,25 @@ int main(int argc, char *argv[])
 		exit(EXIT_FAILURE);
 	}
 
+	if (!strcmp(args.cmd, "delete")) {
+		close(fd);
+
+		if (strcmp(args.type, "ovs")) {
```

Comment:
(I should probably look closer, but)  what's to keep the unprivileged user from deleting any nics he doesn't own?

Reply:
That's introduced later by looking at the DB.